backend/routes.go

```diff
@@ -25,8 +25,10 @@ func (s *server) handleRoutes() error {
 
 	router.HandleFunc("/users/", s.handleAddUser()).Methods("POST")
 	router.HandleFunc("/users/{id}", s.handleGetUser()).Methods("GET")
+	router.HandleFunc("/articles", s.handleGetArticles()).Methods("GET")
 	router.HandleFunc("/articles/", s.handleAddArticle()).Methods("POST")
 	router.HandleFunc("/articles/{id}", s.handleGetArticle()).Methods("GET")
+	router.HandleFunc("/articles/{id}", s.handleAddVote()).Methods("PUT")
```

Comment:
The second route involves voting, maybe the route should be named different for clarity. Thoughts?

Reply:
if youre not using the ID then you don't need `{id}`